torch/ao/quantization/fx/prepare.py

```diff
@@ -115,6 +115,8 @@
 # list of dtypes to not add observers to
 _DO_NOT_OBS_DTYPE_LIST = [int, float, torch.bool, None]
 
+_DEFAULT_FP32_OBS_OR_FQ_CTR = PlaceholderObserver.with_args(dtype=torch.float)
```

Comment:
Does this also work as FQ?

Reply:
yeah this works for FQ as well, it is not inserted to the graph, only the dtype here is used